File: read_cv.py
import pypdf
reader = pypdf.PdfReader("dist/assets/General-CV(Sunny')-BnyBSLHl.pdf")
with open("cv_text.txt", "w", encoding="utf-8") as f:
    for page in reader.pages:
        f.write(page.extract_text() + "\n")

File: read-cv.mjs
import fs from 'fs';
import pdfParse from 'pdf-parse';

const pdfFile = fs.readFileSync("dist/assets/General-CV(Sunny')-BnyBSLHl.pdf");

pdfParse(pdfFile).then(function(data) {
    console.log(data.text);
});
